index.js

```diff
@@ -0,0 +1,58 @@
+
+let hoursHand = document.getElementById('hours');
+let minutesHand = document.getElementById('minutes');
+let secondsHand = document.getElementById('seconds');
+
+let dateLabel = document.getElementById('date-label');
+let timezoneSelector = document.getElementById("timezone");
+
+//Documentatia: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf
+let timeZones = Intl.supportedValuesOf('timeZone');
```

Comment:
Dacă nu resetezi variabila, o poți declara folosind `const`.

```suggestion
const hoursHand = document.getElementById('hours');
const minutesHand = document.getElementById('minutes');
const secondsHand = document.getElementById('seconds');

const dateLabel = document.getElementById('date-label');
const timezoneSelector = document.getElementById("timezone");

//Documentatia: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf
const timeZones = Intl.supportedValuesOf('timeZone');
```

Reply:
Ai schimbat unde am indicat, și câteva alte locuri, dar nu peste tot.

Regula este: Punem `const` mereu, și doar când suntem obligați punem `let`.